engine/apps/alerts/tasks/create_contact_points_for_datasource.py

```diff
@@ -18,22 +17,23 @@ def get_cache_key_create_contact_points_for_datasource(alert_receive_channel_id)
 
 
 @shared_dedicated_queue_retry_task
-def schedule_create_contact_points_for_datasource(alert_receive_channel_id, datasource_list):
+def schedule_create_contact_points_for_datasource(alert_receive_channel_id, datasource_list, retry_counter=0):
```

Comment:
I guess we have this custom retry handling (instead of using celery retries) because a retry in the create task is really triggering the schedule task (ie. a different task) and we want to keep the count of tasks since the initially scheduled create task, right? Maybe adding some comment would be useful.